depthai_sdk/src/depthai_sdk/oak_camera.py

```diff
@@ -131,14 +131,21 @@ def camera(self,
             encode (bool/str/Profile): Whether we want to enable video encoding (accessible via cameraComponent.out_encoded). If True, it will use MJPEG
             name (str): Name used to identify the X-out stream. This name will also be associated with the frame in the callback function.
         """
-        socket = source
         if isinstance(source, str):
-            socket = parse_camera_socket(source.split(",")[0])
+            socket_name = source.split(",")[0].lower()
```

Comment:
And here you replace source, so it can be only `dai.CameraBoardSocket` type. But how user can specify socket with type, like `cam_b,c`?